packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -723,8 +724,8 @@ class ScrollableBaseNative extends Component {
 		}
 
 		if (this.props.rtl && canScrollHorizontally) {
-			/* FIXME: RTL / this calculation only works for Chrome */
-			scrollLeft = bounds.maxLeft - scrollLeft;
+			/* FIXME: RTL / this calculation only works for Chrome and Safari */
```

Comment:
It looks better to remove this comment since this is not so helpful to framework devs anymore.